examples/resource_manager/deployment-yaml.yaml

```diff
@@ -152,13 +153,34 @@ inputs:
   image_sku:
     default: 16.04.0-LTS
 
+  endpoint_resource:
+    default: https://management.core.windows.net/
+
+  endpoints_resource_manager:
+    default: https://management.azure.com
```

Comment:
If someone uses Azure Stack, this is what they provide? What if they use Azure (no Azure Stack)?

Reply:
Default values is fully same as used before, and moved from constants to plugin.yaml